extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -62,18 +62,25 @@ void FuzzyDuck::FuzzAllFunctions() {
 	EndFuzzing();
 }
 
-string FuzzyDuck::GenerateQuery() {
-	LogTask("Generating query with seed " + to_string(seed));
+idx_t FuzzyDuck::GenerateRandomNuber() {
 	auto &engine = RandomEngine::Get(context);
-	// set the seed
 	engine.SetSeed(seed);
-	// get the next seed
 	seed = engine.NextRandomInteger();
+	return seed;
+}
+
+string FuzzyDuck::GenerateQuery(idx_t number_of_statements) {
+	LogTask("Generating query with seed " + to_string(seed));
```

Comment:
Maybe you can change this to say "Generating multi statement query with seed {seed}".
Or split it up so that it either creates the log that one query is being generated or a multi-statement query is being generated